runtime/commonTest/src/kotlinx/io/ByteBufferTest.kt

```diff
@@ -1,78 +0,0 @@
-/*
- * Copyright 2019 JetBrains s.r.o.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package kotlinx.io
-
-import kotlinx.serialization.internal.HexConverter
-import kotlinx.serialization.test.shouldBe
-import kotlin.test.Test
-import kotlin.test.assertEquals
-
-class ByteBufferTest {
-    @Test
-    fun longsAndDoubles() {
```

Comment:
I'd suggest deleting tests with deleting a class, not only because its deprecated

Reply:
Usually, it's a good policy, but here it's mostly about aliases that are going to be removed in the next release. And we are not afraid of breaking anything here (as it is a standalone code)